platform-metrics/build.gradle.kts

```diff
@@ -19,13 +19,13 @@ dependencies {
 
   implementation("io.github.mweirauch:micrometer-jvm-extras:0.2.2")
   implementation("org.slf4j:slf4j-api:1.7.36")
-  implementation("org.apache.logging.log4j:log4j-slf4j-impl:2.17.1")
+  implementation("org.apache.logging.log4j:log4j-slf4j-impl:2.19.0")
```

Comment:
@laxmanchekka The specific logging implementation is not provided by the framework - that's the responsibility of each individual container to select and provide their own implementation. I believe this was a recently introduced regression from one of your PRs - can you follow up and fix (for example, providing the log4j metric conditionally, or splitting it out for consumers to optionally provide).

Reply:
https://github.com/hypertrace/service-framework/pull/60 - PR that introduced issue